src/prefix.rs

```diff
@@ -439,3 +455,9 @@ mod test {
         assert_eq!(res.unwrap().as_slice(), &[]);
     }
 }
+
+#[test]
+fn prefix_debug() {
+    let prefix: Prefix<String, String> = Prefix::new(b"lol", &[Key::Val8([8; 1])]);
+    dbg!(prefix);
```

Comment:
Could you test an actual string representation? Similar to this:

```
assert_eq!(format!("{:?}", binary), "Binary(0735aacb00ff)");
```

Then you can remove `dbg!` here to avoid the debug output in the test